cassandra/cqlengine/columns.py

```diff
@@ -168,6 +168,36 @@ def __init__(self,
         self.position = Column.instance_counter
         Column.instance_counter += 1
 
+    def __ne__(self, other):
+        if isinstance(other, Column):
+            return self.instance_counter != other.instance_counter
+        return NotImplemented
+
+    def __eq__(self, other):
+        if isinstance(other, Column):
+            return self.instance_counter == other.instance_counter
+        return NotImplemented
+
+    def __lt__(self, other):
+        if isinstance(other, Column):
+            return self.instance_counter < other.instance_counter
+        return NotImplemented
+
+    def __le__(self, other):
+        if isinstance(other, Column):
```

Comment:
That is what I meant, but now that you write it out, it doesn't seem to buy much. I guess I'm ambivalent on that.